WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -470,4 +493,42 @@ class PostsListActivity : AppCompatActivity(),
     private fun updateMenuTitle(title: UiString, menuItem: MenuItem): MenuItem? {
         return menuItem.setTitle(uiHelpers.getTextOfUiString(this@PostsListActivity, title))
     }
+
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    @SuppressWarnings("unused")
+    fun onEvent(event: QuickStartEvent) {
```

Comment:
Could we maybe move this whole part (subscription, and all the rest) to the view model? And we could use the SnackbarSequencer to show the snackbar?

Reply:
Yep I think we could move to the ViewModel. Currently I used the same approach and structure used in the `PagesFragment` for the `CREATE_NEW_PAGE` tutorial so I think if we change here we should change there also but maybe this is out of the scope of this PR? Let me know wdyt, if you think it's worth to modify maybe we can open an issue to come back to it in a dedicated PR I guess (not sure we can have it within this code freeze but since it is not a functional issue at least we trace it).

PS: about if to use the snackbar sequencer, I was just writing `Yep! Good idea!` but then realized that here we have a `WPDialogSnackbar` and not the `WPSnackbar` we use in the sequencer and all the QuickStart is using the `WPDialogSnackbar`. At first sight they seem similar with some difference in styling (?), so I guess we could extend the sequencer to manage them also but again maybe it's a wider scope 🤷‍♂️.

Reply:
That makes sense 👍. We should definitely merge the two snackbars and move all the logic to view models but let's do it in a separate PR.